ui/oopsyraidsy/oopsyraidsy_config.js

```diff
@@ -222,6 +222,14 @@ UserConfig.registerOptions('oopsyraidsy', {
       type: 'checkbox',
       debugOnly: true,
     },
+    {
+      id: 'ClickToCopy',
+      name: {
+        en: 'Enable click-to-copy',
+        ko: '클릭해서 복사하기 활성화',
+      },
+      type: 'checkbox',
```

Comment:
Can you add a default value here?   What do you think about defaulting this to `true` so that people have a way to turn it off, but it's on by default?

(For what it's worth, I am not sure this needs an option, since it requires a user interaction to do the copy.  It seems unlikely that people will be doing something that requires and copying and pasting while raiding and then click on oopsy in the middle of that operation.  If you think we need an option, then I think this is fine.  I just am a little skeptical.)

Reply:
okay, I was thinking about what is better. I'll go for no option.